### ISSUE TITLE
pys-onClick is deprecated and broken, should be ultimately removed

### ISSUE THREAD

**Issue by madhur-tandon** [pyscript/pyscript#1213]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help

### What happened?

There are tests that check for `pys-onClick` which is deprecated, we should change the tests to use `py-click` instead and eventually remove the tests that check for its deprecation.

The two tests are:
- test_pys_onClick_shows_deprecation_warning
- test_splashscreen_closes_on_error_with_pys_onClick

Tagging @FabioRosado for more context

**Comment by FabioRosado** [comment]
I think since we still have the elements we should test them then when we remove these attributes we remove the tests? 🤔

**Comment by madhur-tandon** [comment]
The issue is basically that these tests fail and are kinda written incorrectly. For example, the `id` attribute is passed as `"1"` which isn't allowed according to the HTML specification. If we fix the `id` stuff, then also the test fails but with a different reason. Hence, @antocuni suggested to open an issue about it (aka this) which is also referenced in the PR we closed yesterday regarding wrapping `runPython` in async -- since we marked those 2 tests as skipped.
